examples/discord_links.py

```diff
@@ -3,7 +3,9 @@
 
 from coc.ext import discordlinks
 
-client = discordlinks.login(os.environ["LINKS_API_USERNAME"], os.environ["LINKS_API_PASSWORD"])
+client = discordlinks.login(
```

Comment:
Not raising an error if the environmental values are not set leads to ambiguous errors down the track, so if you could revert this back to `os.environ["..."]` that would be great 👍